icgcget/clients/download_client.py

```diff
@@ -10,15 +32,20 @@ class DownloadClient(object):
     __metaclass__ = abc.ABCMeta
 
     @abc.abstractmethod
-    def __init__(self, json_path=None):
+    def __init__(self, json_path=None, docker=False, log_dir=None):
+
         self.logger = logging.getLogger('__log__')
         self.jobs = []
         self.session = {}
+        self.log_dir = log_dir
         self.path = json_path
+        self.docker = docker
         self.repo = ''
+        self.docker_mnt = '/icgc/mnt'
+        self.docker_version = 'icgc/icgc-get:0.3.1'
```

Comment:
This isn't really the docker version per se. Its really about the container. Also, I would have `0.3.1` passed in as a `container_tag` constructor argument. That way, you can centralize this part in a higher context, which is likely to change much more often than the repository name (`icgc/icgc-get`).